src/tracer.ts

```diff
@@ -52,6 +55,60 @@ export class Tracer {
     // OpenTracing API methods
     // ---------------------------------------------------------------------- //
 
+    /**
+     * Gets the currently active span.
+     *
+     * @return the active Span. This is a shorthand for tracer.spanManager().active().
+     */
+    activeSpan(): Span | null {
+        return this.spanManager().active();
+    }
+
+    /**
+     * Starts a span, activates the span, executes the function, and finishes the span.
+     *
+     * @param {string} name the name of the operation.
+     * @param {SpanOptions} options options for the newly created span.
+     * @param {Function} f the function to execute during span activation.
+     * @return {A} the return value of the executed function.
+     * @template A
+     */
+    runSpan<A>(name: string, options: SpanOptions, f: () => A): A {
```

Comment:
This name sounds a bit weird to me, as this method doesn't run a span.

Reply:
Other suggestion?

It starts and stops (runs?) the span.
It runs the function, with a new active span for its duration.

Reply:
I think I would actually remove it completely for the first version, to keep the API surface as small as possible. Then more helpers can be added over time if necessary and discussed individually.